.github/workflows/release.yml

```diff
@@ -9,14 +9,11 @@ on:
     tags:
       - '*'
 
-permissions:
-  contents: read # minimal required permissions to clone repo
-
 jobs:
   release:
     name: Release
     permissions:
-      contents: write
+      contents: read
```

Comment:
I prefer to set the permissions on the global level. Otherwise you need to set it again for each new job (and people will probably forget it)